packages/babel-parser/test/fixtures/esprima/invalid-syntax/migrated_0086/input.js

```diff
@@ -1 +0,0 @@
-(a, (b)) => 42
```

Comment:
This test is removed because it is almost identical to https://github.com/babel/babel/blob/4eecb1b3f5476bb85c9b4e2a80306b2409811aa5/packages/babel-parser/test/fixtures/es2015/arrow-functions/inner-parens-2/input.js#L1